contracts/VaultV3.vy

```diff
@@ -700,13 +706,13 @@ def _redeem(
     # funds from strategies.
     if requested_assets > curr_total_idle:
 
-        # Cache the input withdrawal queue.
-        _strategies: DynArray[address, MAX_QUEUE] = strategies
+        # Cache the default queue.
+        _strategies: DynArray[address, MAX_QUEUE] = self.default_queue
```

Comment:
Copying from storage by default? Good idea?

Reply:
I think given the expectation that it will almost always use the default_queue its worth it